spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/logging/LogFileWebEndpointAutoConfiguration.java

```diff
@@ -60,16 +62,25 @@ public LogFileWebEndpoint logFileWebEndpoint(Environment environment) {
 		public ConditionOutcome getMatchOutcome(ConditionContext context,
 				AnnotatedTypeMetadata metadata) {
 			Environment environment = context.getEnvironment();
-			String config = environment.resolvePlaceholders("${logging.file.name:}");
+			String config = environment
+					.resolvePlaceholders("${" + LogFile.FILE_PROPERTY + ":}");
+			if (!StringUtils.hasText(config)) {
+				config = environment.resolvePlaceholders(
+						"${" + LogFile.DEPRECATED_FILE_PROPERTY + ":}");
+			}
```

Comment:
Here I changed `logging.file.name` for `LogFile.DEPRECATED_FILE_PROPERTY` because that field is annotated with `@Deprecated`, which will make it easier to find and remove the deprecated code caused by this PR when the deprecation phase is over.